apps/allocations/app/store/account.js

```diff
@@ -54,8 +59,7 @@ const loadAccountData = async accountId => {
     // TODO: Should we standarize the naming and switch to getAccount instead of getPayout?
     app
       .call('getPayout', accountId)
-      .first()
-      .map()
+      .pipe(first(), map())
```

Comment:
I went to check if this empty map could be removed and as far as I can tell this entire function is actually unused now.

Reply:
Cool will take out